code/modules/overmap/helm.dm

```diff
@@ -194,6 +194,9 @@
 			if(!new_name)
 				return
 			new_name = trim(new_name)
+			var/prefix = current_ship.source_template.prefix
+			if (!(findtext(new_name, "KOS") || findtext(new_name, prefix)))
+				new_name = addtext(prefix, " ", new_name)
```

Comment:
I don't like this, why use addtext? That's an OLD byond proc, that isn't used like anywhere else in the codebase as far as I can tell. Simple string concatenation will work fine here:
```suggestion
				new_name = prefix + " " + new_name
```

Reply:
yeah `new_name = "[prefix] [new_name]"` would also work